### PR TITLE
#156882163 Redesign gaming screen

### PR DESCRIPTION
**Pull request by emmabaye**
#### What does this PR do?
Redesign of the gaming screen.
#### Description of Task to be completed?
N/A
#### How should this be manually tested?
Install dependencies and start the application. Open app in  the
 browser and click on `play with guests` or sign in and play
 with friends or strangers.
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#156882163
#### Screenshots (if appropriate)
firefox_screenshot_2018-05-15t12-06-40 984z

#### Questions:
N/A

### CODE REVIEW

File: public/views/app.html [hunk 1]
```diff
@@ -1,87 +1,177 @@
...
-      <div id="menu-timeremaining-container">
-        <div id="menu-container">
-          <span id="menu-button">Menu</span>
+        
+
+      <div landing class="h-container" >
+  
+  
+        <div class="logo" ng-controller="IndexController">
+          <img src="/img/playing-cards.png" alt="">
```
Comment (Orlayhemmy) [inline public/views/app.html:27]: There are too many uncovered lines. Always ensure you don't leave lines uncovered except when you need to differentiate a section from another, it makes your code look better.

**Review by Orlayhemmy — COMMENTED** [review]
Good job fixing the angular error and also creating the gaming screen

**Comment by chuzksy-codeactive** [comment]
You have put in lots of effort into this task, I admire your attitude to get this task done. But I think you should have a look why your build is failing. Job well done.

**Comment by chuzksy-codeactive** [comment]
Good job. You have put a great energy into this task. Keep up the good work

**Pull request merged**